test/ibmq/test_proxies.py

```diff
@@ -37,15 +35,15 @@
 INVALID_ADDRESS_PROXIES = {'https': '{}:{}'.format('invalid', PORT)}
 
 
-@skipIf(sys.version_info >= (3, 7), 'pproxy version not supported in 3.7')
+@skipIf(python_version < (2, 7), 'pproxy is supported only on or above python 2.7')
```

Comment:
Slightly related to the comment on the .txt file, and to the last comment in the issue - the idea was to keep skipping if an invalid combination was found, ie. something similar to:
* Python>3.5 and pproxy==1.2.2
* Python==3.5 and pproxy > 1.2.2

Can you update the `skipIf` accordingly?